### DISCUSSION TITLE
Subsidence in Oil Field

### DISCUSSION THREAD

**Discussion by Obaida777** [gmtsar/gmtsar#465] [Q&A]
Hello Everyone 
I hope All doing well. 

Kindly if my Study Area is bare desert so what is the best Unwrap threshold=? And Smoothing Factor =? In GMTSAR?

Best Regards

**Comment by Xiaohua-Eric-Xu** [comment]
There are a few kind of deserts. If you mean somewhere like the Majove, you could unwrap everything with a very small threshold like 0.05. But if you mean desert with sands, then it easily decorrelates and maybe you have to avoid unwrapping over such areas using some masking.

**Reply by Obaida777** [reply]
Thanks

**Reply by Obaida777** [reply]
Kindly, what about smooching factor ?

**Reply by Xiaohua-Eric-Xu** [reply]
Anything between 0.1 to 10 should be OK for the sbas code in GMTSAR.
